test/receiver.spec.ts

```diff
@@ -40,6 +40,61 @@ describe("Receiver", () => {
     assert.isFalse(receiver.isOpen(), "Receiver should not be open.");
   });
 
+  it(".remove() removes event listeners", async () => {
+    const receiver = await connection.createReceiver();
+    receiver.on(rhea.ReceiverEvents.receiverOpen, () => {
+      /** no-op */
+    });
+
+    assert.isTrue(receiver.isOpen(), "Receiver should be open.");
+    assert.isAtLeast(receiver.listenerCount(rhea.ReceiverEvents.receiverOpen), 1);
+
+    receiver.remove();
+
+    // TODO: receiver.isOpen() returns true at this point, need to find out why
```

Comment:
`receiver.remove()` just causes references to the receiver to be removed from the rhea `Session`, but doesn't actually close the link. We'd still need to explicitly close it.

rhea doc comments actually say `close` should be called on active sessions/connections instead of remove:
https://github.com/amqp/rhea/blob/5fbcee452afd8888f276a1566e87fe26530571d3/lib/link.js#L93-L97

Reply:
got it, removed the TODO in https://github.com/amqp/rhea-promise/pull/63/commits/7e051c49a8f3a2780cd93adb2f2ae9448b78f6bf